### ISSUE TITLE
Edit Account button under Administrator controls not working [Blocked on UX design]

### ISSUE THREAD

**Issue by srushtirk** [oppia/oppia-android#3825]
**Describe the bug**
Edit Account button under Administrator controls doesn't work.
Note: This button doesn't work in the UI Mocks as well: https://xd.adobe.com/view/3dca36c2-5115-419c-b25e-0f10526b077c-6899/screen/54511d86-eeca-4bf9-a891-22c0a6aa9a08/

**To Reproduce**
1. Go to Oppia app
2. On 'Select your profile' page, click on administrator controls on the bottom right
3. Click on 'Edit Account' button - See error

**Expected behavior**
Edit Account button should be directed to edit account options/dialog box.

**Demonstration**
![image](https://user-images.githubusercontent.com/27875087/134093420-4a7eea47-ec1c-4706-ac76-eafd60346608.png)

**Environment**
- Device/emulator being used: TBD
- Android or SDK version (e.g. Android 5 or SDK 21): TBD
- App version: TBD

**Additional context**
The device language was selected as Portuguese

**Comment by BenHenning** [comment]
I believe this isn't a blocker for MR3 (& am thus removing it from our blocker list). If I got that wrong, please re-add it @srushtirk.

Also, this is an existing issue in the app, not a regression.

**Comment by coder2699** [comment]
@srushtirk do we have an existing activity/fragment for the same & only linking part is left? or this needs to be implemented from scratch?

**Comment by BenHenning** [comment]
Good question @coder2699. We're actually currently discussing this as part of the design of trying to figure out how to present the account's phone number/email address (once those are included in the app). This is an ongoing discussion with @mschanteltc that needs to be determined before we can make progress on this issue.

**Comment by rishidyno** [comment]
@BenHenning I am also interested in working on this issue. Also, @coder2699  I looked up any implementations that might have existed but I can't seem to find any. so I think it has to be implemented from scratch.

**Comment by BenHenning** [comment]
@rishidyno just to clarify, are you interested in introducing the new UI? I can cc you on the ongoing email thread so that you can get the necessary context.

**Comment by rishidyno** [comment]
Yes Definitely! @BenHenning I would love to work on introducing new ui.

**Comment by BenHenning** [comment]
@rishidyno either Rajat or I will follow up here once the mocks are ready (looking back on the email thread, I think the product questions have been resolved; we're just waiting on the mocks to be finished).

**Comment by rishidyno** [comment]
Ok! Thanks, @BenHenning Once the mocks are ready I will start working on them.

**Comment by rishidyno** [comment]
@BenHenning @rt4914 any progress on the mocks?

**Comment by rt4914** [comment]
@rishidyno Currently we have decided to hide the feature which is getting done in #4006

**Comment by BenHenning** [comment]
For searching context, this issue was found in 0.6-alpha (MR3).

**Comment by BenHenning** [comment]
Per Rajat's comment, this is a duplicate of #4006.
